xml/admin_nfsganesha.xml

```diff
@@ -745,17 +665,16 @@ ganesha_configurations:
   <title>Starting or Restarting &ganesha;</title>
 
   <para>
-   To enable and start the &ganesha; service, run:
+   To start the &ganesha; service, run:
   </para>
 
-<screen>&prompt.sminion;<command>systemctl</command> enable nfs-ganesha
-&prompt.sminion;<command>systemctl</command> start nfs-ganesha</screen>
+<screen>&prompt.cephuser;ceph orch start nfs.<replaceable>SERVICE_ID</replaceable></screen>
 
   <para>
-   Restart &ganesha; with:
+   To restart the &ganesha; service, run:
   </para>
 
-<screen>&prompt.sminion;<command>systemctl</command> restart nfs-ganesha</screen>
+<screen>&prompt.cephuser;ceph orch restart nfs.<replaceable>SERVICE_ID</replaceable></screen>
```

Comment:
The `systemctl` examples would start/restart a single NFS ganesha daemon, but `ceph orch restart..` will restart the entire NFS service (and all the containing daemons)... 

should we use `ceph orch daemon ...` instead? 

or maybe reference the generic section on this topic? 
https://susedoc.github.io/doc-ses/master/single-html/ses-admin/#cha-ceph-operating-individual